cli/src/main/kotlin/org/partiql/shell/Shell.kt

```diff
@@ -36,6 +36,7 @@ import org.partiql.lang.eval.ExprValue
 import org.partiql.lang.eval.ExprValueFactory
 import org.partiql.lang.eval.PartiQLResult
 import org.partiql.lang.eval.delegate
+import org.partiql.lang.syntax.PartiQLParserBuilder
```

Comment:
Think we should also update the [CHANGELOG.md](https://github.com/partiql/partiql-lang-kotlin/blob/c49c846cf6395928401c7e76f11d64b3432183bf/CHANGELOG.md) w/ this fix

Reply:
My bad. `!!` was removed in #883, which was unreleased. Updating the CHANGELOG isn't necessary.